### PR TITLE
Translates SharedExamples to pt-BR

### PR DESCRIPTION
**Pull request by pedrovereza**
Another small step on #610 

Special trouble to translate "that's the way the cookie crumbles!" 😄

### CODE REVIEW

**Comment by pedrovereza** [comment]
@marciok 😉

File: Documentation/pt-br/SharedExamples.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+import Quick
+
+sharedExamples("everything under the sea") {
+  // ...
+}
+
+itBehavesLike("everything under the sea")
+```
+
+> Em Objective-C, é necessário passar um bloco que recebe um `QCKDSLSharedExampleContext, mesmo se esse argumento não for usado. Desculpe, mas é assim que a banda toca! :trumpet: :notes:
```
Comment (morganchen12) [inline Documentation/pt-br/SharedExamples.md]: Missing backtick after `QCKDSLSharedExampleContext`.

I'm not qualified to review the rest of the doc, though :)

Comment (pedrovereza) [inline Documentation/pt-br/SharedExamples.md]: Good catch!

**Comment by modocache** [comment]
Fantastic! And thanks for spotting that, @morganchen12. :)

@pedrovereza do you know of anyone who can team up with you to review these? I don't speak the language. 😅

**Comment by pedrovereza** [comment]
@modocache normally @marciok reviews the translations to pt-BR. But I can try to find someone else if he's not available to review future translations :wink:

**Comment by marciok** [comment]
@modocache @pedrovereza  I'm on it! Sorry that I was absent for some time. I'm going to review tomorrow 😀

**Comment by modocache** [comment]
Ah, thank you! Sorry, that slipped my mind :)

Thanks for all the translations! 💓

**Comment by marciok** [comment]
Great job @pedrovereza, I'll try to translate another doc within the next weeks. Thanks for the translation 👍

**Pull request merged**